packages/frontend/src/hooks/useCalculateTotal.ts

```diff
@@ -119,23 +117,12 @@ export const getCalculationColumnFields = (
     itemsMap: Record<string, Field | TableCalculation>,
 ) => {
     //This method will return the metric ids that need to be calculated in the backend
-    // We exclude metrics we already calculate
-    const numericTypes: string[] = [
-        MetricType.NUMBER,
-        MetricType.COUNT,
-        MetricType.SUM,
-    ]; // We calculate these types already in the frontend
 
     const items = selectedItemIds
         ?.map((item) => {
             return itemsMap[item];
         })
-        .filter(
-            (item) =>
-                isField(item) &&
-                isMetric(item) &&
-                !numericTypes.includes(item.type.toString()),
-        );
+        .filter((item) => isField(item));
```

Comment:
We removed `isMetric()` on purpose? 
Now it makes a request even if there are only dimensions. 
We can also move this logic inside the reduce and have a single loop and simplify the code. 

<img width="1455" alt="Screenshot 2023-11-24 at 09 18 08" src="https://github.com/lightdash/lightdash/assets/9117144/3d41e36c-e108-4ef6-9616-4b3c06513cdd">

Reply:
I did it in purpose, but  perhaps it was not a good idea, I'll put it back